.github/workflows/validate-reference-test-flow.yml

```diff
@@ -0,0 +1,34 @@
+name: "Validate Reference Test Flow"
+on: [pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  validation-reference-test-flow:
+    runs-on: gha-runner-scale-set-ubuntu-22.04-amd64-med
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: get-branch-name
+        id: extract_branch
+        run: |
+          echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+
+      - name: Setup Test Environment
+        uses: ./.github/actions/setup-environment
+        with:
+          go-corset: true
+
+      - name: Run Reference Blockchain test sample
+        run: ./gradlew -Dblockchain=Ethereum referenceBlockchainTests -x spotlessCheck
```

Comment:
Hmmmm, this doesn't look right to me ... it will run all the tests, no?  I think you need e.g. `--tests "BlockchainReferenceTest_55"` where `55` is just one of the tests I picked at random.  Not sure what a good example is, but anyway that might be fine.

Reply:
For reference, `BlockchainReferenceTest_104` is the one which contains the test you chose ... `stChainId`.

```
grep "stChainId" `find * -name "BlockchainReferenceTest_*"`
```